src/modules/bulletproofs/bulletproofs_util.h

```diff
@@ -49,4 +49,17 @@ static void secp256k1_bulletproofs_le64(unsigned char *output, const uint64_t n)
     output[7] = n >> 56;
 }
 
+/* Check if n is power of two*/
+static int secp256k1_check_power_of_two(size_t n) {
+    return n > 0 && (n & (n - 1)) == 0;
```

Comment:
nit: maybe call this `_is_power_of_two`  ... (`check` reminds me of `CHECK` in this library...)